### ISSUE TITLE
As a content editor working on transcriptions, I want the page to autosave existing work in the transcription editor when clicking outside the annotation box.

### ISSUE THREAD

**Issue by kseniaryzhova** [Princeton-CDH/geniza#1263]
**Is your feature request related to a problem? Please describe.**
Related to #1177 - clicking outside the annotation box closes the annotation box without saving changes. 

**Describe the solution you'd like**
We would like the annotation box to autosave any changes made when a user clicks outside the box.

**Comment by blms** [comment]
Adding question label, awaiting MR's feedback on whether this is still needed after #1177 was closed.

**Comment by blms** [comment]
Closing as `wontfix` as #1177 was deemed acceptable, with the caveat that we may need to refine the behavior of transcribing multiple pages on the same document simultaneously.
